### ISSUE TITLE
Remove old batman-adv

### ISSUE THREAD

**Issue by Grotax** [ffsh/ansible#22]
example from slybek
```
sylsbek ➜  ~ dkms status
batman-adv, 2021.2, 5.10.0-10-amd64, x86_64: built
batman-adv, 2021.2, 5.10.0-12-amd64, x86_64: built
batman-adv, 2021.2, 5.10.0-9-amd64, x86_64: built
batman-adv, 2021.3, 5.10.0-12-amd64, x86_64: built
batman-adv, 2021.3, 5.10.0-13-amd64, x86_64: built
batman-adv, 2022.0, 5.10.0-13-amd64, x86_64: built
batman-adv, 2022.0, 5.10.0-14-amd64, x86_64: built
batman-adv, 2022.1, 5.10.0-18-amd64, x86_64: built
batman-adv, 2022.1, 5.10.0-19-amd64, x86_64: installed (WARNING! Diff between built and installed module!)
batman-adv, 2022.2, 5.10.0-18-amd64, x86_64: installed
batman-adv, 2022.2, 5.10.0-19-amd64, x86_64: built
```
```
sylsbek ➜  ~ ls /usr/src 
batman-adv-2021.2  linux-headers-5.10.0-18-amd64
batman-adv-2021.3  linux-headers-5.10.0-18-common
batman-adv-2022.0  linux-headers-5.10.0-19-amd64
batman-adv-2022.1  linux-headers-5.10.0-19-common
batman-adv-2022.2  linux-kbuild-5.10
```

**Comment by Grotax** [comment]
```
([0-9]{4}\.[0-9]{1})
```
to find versions in dkms status

then compare to current version, call via ansible
remove via dkms remove
then remove directories in /usr/src

**Comment by Grotax** [comment]
Cleanup for /opt implemented in 129465286358953b75ad64be379e3f61ca5a838f
Works easily in this case since the directories and archives with the current version are not needed and can be deleted after unpack and install

**Comment by Grotax** [comment]
Solved with:
ac4eb322cf8deb957f92ac607c35ed541b46a15f
5b9a18184efe879eddd330bcd31fc5a4fe032f09
938e428eaf5117c3a541697e8c327358b05ecca9
